shapetrees-schema.shex

```diff
@@ -3,38 +3,51 @@ PREFIX st: <http://www.w3.org/ns/shapetrees#>
 PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
 PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
 
-<#ShapeTree> {
-  a st:ShapeTree ;
+<#ShapeTreeShape> {
+  a [st:ShapeTree] ;
   (
     st:expectsType [st:Container] ;
     st:contains @<#ShapeTree> * ;
```

Comment:
s/ShapeTree/ShapeTreeShape/

Reply:
@ericprud Your regex implies that it should be `ShapeTreeShape`, which is what is included in this commit, so did you mean to say it should stay `ShapeTree`? I added Shape at the end because we've been using that convention for the other shapes but weren't using it for the ShapeTree shape itself.

Reply:
Oops, too terse. st:contains needs to reference @<#ShapeTreeShape>

Reply:
Ahhhh of course.

```suggestion
    st:contains @<#ShapeTreeShape> * ;
```